sklearn/ensemble/gradient_boosting.py

```diff
@@ -207,13 +262,57 @@ def predict(self, X):
 
 
 class ScaledLogOddsEstimator(LogOddsEstimator):
-    """Log odds ratio scaled by 0.5 -- for exponential loss. """
+    """Log odds ratio scaled by 0.5 -- for exponential loss.
+
+    This estimator takes only the binary target labels of the training set into
+    account. The prediction is independent of the X values in the training set
+    and those passed to the :func:`predict` function.
+
+    It is like :class:`LogOddsEstimator` but takes the log of the *square* of
+    the odds ratio.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.ensemble import gradient_boosting
+    >>> a = np.random.rand(4, 2)
+    >>> y = np.array([1, 1, 0, 1])
+    >>> scaled = gradient_boosting.ScaledLogOddsEstimator()
+    >>> unscaled = gradient_boosting.LogOddsEstimator()
+    >>> scaled.fit(a, y)
+    >>> unscaled.fit(a, y)
+    >>> print(0.5 * unscaled.prior, scaled.prior)  # doctest +ELLIPSIS
```

Comment:
only one space after `)`

Reply:
This was in accordance with [PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments): "Inline comments should be separated by at least two spaces from the statement."

Should I delete the double space?

Reply:
nope, if that's the case, I was wrong. PEP8 most definitely trumps me, lol.